app/src/main/java/com/keylesspalace/tusky/receiver/SendStatusBroadcastReceiver.kt

```diff
@@ -114,25 +113,25 @@ class SendStatusBroadcastReceiver : BroadcastReceiver() {
 
                 context.startService(sendIntent)
 
-                val builder = NotificationCompat.Builder(
+                // Notifications with remote input active can't be cancelled, so let's replace it with another one that will dismiss automatically
+                val notification = NotificationCompat.Builder(
                     context,
                     NotificationHelper.CHANNEL_MENTION + senderIdentifier
                 )
                     .setSmallIcon(R.drawable.ic_notify)
                     .setColor(context.getColor(R.color.notification_color))
                     .setGroup(senderFullName)
-                    .setDefaults(0) // So it doesn't ring twice, notify only in Target callback
-
-                builder.setContentTitle(context.getString(R.string.post_sent))
-                builder.setContentText(context.getString(R.string.post_sent_long))
-
-                builder.setSubText(senderFullName)
-                builder.setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
-                builder.setCategory(NotificationCompat.CATEGORY_SOCIAL)
-                builder.setOnlyAlertOnce(true)
-
-                // There is a separate "I am sending" notification, so simply remove the handled one.
-                notificationManager.cancel(notificationId)
+                    .setDefaults(0) // We don't want this to make any sound or vibration
+                    .setOnlyAlertOnce(true)
+                    .setContentTitle(context.getString(R.string.reply_sending))
+                    .setContentText(context.getString(R.string.reply_sending_long))
+                    .setSubText(senderFullName)
+                    .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+                    .setCategory(NotificationCompat.CATEGORY_SOCIAL)
+                    .setTimeoutAfter(5000)
+                    .build()
```

Comment:
Should this maybe be extracted?

Reply:
I was thinking about it, it is similar to other places where we build notifications, but not enough to be easily extracted. I think its easier to read like it is.